frontend/src/app/components/MobileNavigationMenu/MobileLogo/MobileLogo.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import Image from "next/image";
+
+import './MobileLogo.scss';
+
+const MobileLogo: React.FC = () => {
+    return (
+        <div className="mobileLogoContainer">
+            <Image src="/Logo.png" alt="Logo of app" width={175} height={146}/>
```

Comment:
Is this path - /Logo.png a correct one? It just hit me into my face but it could be correct, I'm just asking.

Reply:
Yep, images are in the ```/public``` folder, so you make reference on the files just with ```/```.